luminoth/tools/dataset/transform.py

```diff
@@ -30,16 +30,17 @@ def get_output_subfolder(only_classes, only_images, limit_examples,
 @click.option('--data-dir', help='Where to locate the original data.')
 @click.option('--output-dir', help='Where to save the transformed data.')
 @click.option('splits', '--split', required=True, multiple=True, help='Which splits to transform.')  # noqa
-@click.option('--only-classes', help='Whitelist of classes.')
+@click.option('--only-classes', multiple=True, help='Whitelist of classes.')
+@click.option('--merge-classes', help='Merge all classes into a single class')
```

Comment:
Can't we name it `--single-class` or `--merge-all` to make the fact that only a single class creatd more explicit?